src/splitwisesdk/core/Group.java

```diff
@@ -0,0 +1,20 @@
+package splitwisesdk.core;
+
+import java.util.ArrayList;
+
+public class Group {
+
+    public String id;
+    public ArrayList<GroupMember> members = new  ArrayList<>();
+    public ArrayList<Expense> expenses = new ArrayList<>();
+
+
```

Comment:
add method to
boolean addGroupMember(GroupMember groupMember) {
   // Verify that groupMember is not already in members
   // If not then add it to members and return true
   // else do nothing and return false
}

Reply:
public boolean addPeople(People people) {
    return addGroupMember(new GroupMember(people,groupId));
}